MAUI/Cartesian-Charts/Zooming-and-panning.md

```diff
@@ -79,6 +79,43 @@ chart.ZoomPanBehavior = zooming;
 
 {% endtabs %}
 
+### Directional Zooming
+
+The directional Zooming feature enhances your zooming experience by allowing you to zoom in and out in a specific direction. This feature is enabled by setting the [EnableDirectionalZooming]() property to `true` as shown in the following code sample and the [EnablePinchZooming](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartZoomPanBehavior.html#Syncfusion_Maui_Charts_ChartZoomPanBehavior_EnablePinchZooming) should not be set as `false,` because it relies on the pinch gesture direction. The default value of this property is false.
```

Comment:
EnablePinchZooming property should be set as true. Update it in tips

Reply:
Update the changes and add it in tips.